lookout/style/format/descriptions.py

```diff
@@ -184,3 +185,48 @@ def describe_rule_parts_ordinal(feature: OrdinalFeature, name: str,
     """
     cmp, threshold, _ = splits[0]
     return _format_int(cmp, threshold, name)
+
+
+def get_error_description(vnode: VirtualNode, y_pred: int) -> str:
+    """
+    Return the comment with regard to the correct node class.
+
+    :param vnode: Node to describe.
+    :param y_pred: The index of the correct node class.
+    :return: String comment.
+    """
+    column = vnode.start.col
+    if y_pred == CLASS_INDEX[CLS_NOOP]:
+        return "%s at column %d should be removed." % (CLASS_REPRESENTATIONS[vnode.y], column)
+    elif vnode.y == CLASS_INDEX[CLS_NOOP]:
+        return "%s should be inserted at column %d." % (CLASS_REPRESENTATIONS[y_pred], column)
+    else:
+        return "Replace %s with %s at column %d." % (
+            CLASS_REPRESENTATIONS[vnode.y], CLASS_REPRESENTATIONS[y_pred], column)
+
+
+def get_code_chunk(lang: str, code_lines: Sequence[str], line_number: int) -> str:
+    """
+    Return nice code snippet that can be inserted to github message.
+
+    :param lang: Code language.
+    :param code_lines: Code lines.
```

Comment:
This is a bad parameter description, Captain Obvious style :)

> Code split into lines, newline characters included/excluded (idk - please choose)

Regarding `lang`. We've got two naming styles: JavaScript and javascript. Which is used here?